titus-common/src/test/java/com/netflix/titus/common/util/archaius2/Archaius2ObjectConfigurationResolverTest.java

```diff
@@ -68,6 +68,16 @@ public void testDynamicUpdates() {
         assertThat(resolver.resolve(new MyObject("a")).getLimit()).isEqualTo(100);
     }
 
+    @Test
+    public void testBadPattern() {
+        configuration.setProperty("object.a.pattern", "a");
+        configuration.setProperty("object.a.limit", "200");
+        assertThat(resolver.resolve(new MyObject("a")).getLimit()).isEqualTo(200);
+
+        configuration.setProperty("object.a.pattern", "*"); // bad regexp
+        assertThat(resolver.resolve(new MyObject("a")).getLimit()).isEqualTo(200);
```

Comment:
do we need a delay here to make sure Archaius had a chance to propagate the update?

Reply:
In test it is not Archaius but `DefaultSettableConfig`. Update is done in the same thread.